resources/views/users/research/show.blade.php

```diff
@@ -0,0 +1,72 @@
+@extends('layouts.app')
+
+@section('style')
+    <link rel="stylesheet" href="{{ mix('css/users-style.css') }}">
+    <link rel="stylesheet" href="{{ mix('css/modal.css') }}">
+@endsection
+
+@section('content')
+  <div class="container">
+      <div class="row">
+          <div class="col-8">
+            <div class="category">
+                <h2 class="display-5 mb-3">{{ $category->name }}</h2>
+                  @forelse($all_meetings as $meeting)
+                    <div class="category-myroom mx-auto mb-2">
+                      <div class="category-item">
+                        {{ $meeting->date }}<br>{{ $meeting->start_at}} : 00〜
+                      </div>
+                      {{-- <div class="category-item">
+                        <img src="image/begginer.png" alt="">
+                      </div> --}}
+                      <div class="category-item">
+                        {{ $meeting->title}}
+                      </div>
+                      <div class="category-item">
+                        {{-- <a href="{{ route('', ['meeting' => $meeting->id]) }}" class="text-muted"> --}}
```

Comment:
Also remove comments if they're not in use.

Reply:
I use this one. so uncomment it and add route.